### DISCUSSION TITLE
Community

### DISCUSSION THREAD

**Discussion by Ngomana** [sl-nx/superlogin-next#34] [Q&A] [answered]
Is there some sort of discord community which I can join and ask questions on?

**Comment by fynnlyte** [comment, marked as answer]
Nope, this (or stack overflow) would be the right place to ask questions at the moment.

**Reply by fynnlyte** [reply]
By the way - if you're trying out `couch-auth`, better use `npm i @perfood/couch-auth@beta` until 0.17.0 is published because I've been switching out the email template engine

**Reply by Ngomana** [reply]
noted!!!

**Comment by Ngomana** [comment]
By the way, could you please consider creating a discord community for this package?

**Reply by fynnlyte** [reply]
Currently I'm not planning on doing that. Also, I'd like Q&A to be publically available + searchable instead of being located on Discord.

For questions on StackOverflow, feel free to tag them with `couch-auth` (doesn't exist yet, but I'll follow it) and `couchdb`.

If you have general questions about CouchDB, there's a slack/IRC channel. Look at https://couchdb.apache.org/#contribute
